src/store/actions/firestore.ts

```diff
@@ -0,0 +1,16 @@
+import { createAction } from '@reduxjs/toolkit';
+
+import { RequestFirestoreID } from 'app/types';
+import { FirebaseDocument } from 'app/types/firebase';
```

Comment:
let's create a barrel export so you could import `FirebaseDocument` directly from `app/types`